test/functional/Valhalla/src_qtypes/org/openj9/test/lworld/ValueTypeArrayTests.java

```diff
@@ -224,7 +224,7 @@ static void assignDispatch(Object[] arr, int idx, Object src, int arrKind, int s
 	 *
 	 * <p><code>expectedAssignmentExceptions[actualArrayKind][actualSourceKind]</code>
 	 */
-	static Class expectedAssignmentExceptions[][] =
+	static Class[][] expectedAssignmentExceptions =
```

Comment:
While we're tidying up, can you remove the use of raw types too, please (e.g. `Class<?>` instead of just `Class`)?

Reply:
This doesn't appear to have been addressed.